bin/utils/subkey/Cargo.toml

```diff
@@ -17,7 +17,7 @@ rand = "0.7.2"
 clap = "2.33.0"
 tiny-bip39 = "0.7"
 rustc-hex = "2.0.1"
-substrate-bip39 = "0.3.1"
+substrate-bip39 = "0.4.1"
```

Comment:
This won't work, `substrate-bip39` 0.4 uses new schnorrkel which is not yet adapted for substrate properly

Reply:
> not yet adapted for substrate

What does that mean exactly? Do you mean on types? We had some issues with marlin and evm regarding then using different types, but that was all resolved.

Or do you mean the underlying algorithm has changed? If so, where can I read up about that, the crate doesn't have any changelog :( .

Reply:
Ok, seems that schnorrkel get updated to version 0.9.1 (was 0.9.0 recently) which is compatible with current substrate

So I take my words back